library/std/src/io/mod.rs

```diff
@@ -284,7 +284,7 @@ pub(crate) use error::const_io_error;
 mod buffered;
 pub(crate) mod copy;
 mod cursor;
-mod error;
+pub(crate) mod error;
```

Comment:
This is not needed -- you've added it just to access `io::error::const_io_error!`, but that macro is exported above as `pub(crate)`, so you can just use `io::const_io_error!`.